man/systemd.link.xml

```diff
@@ -443,6 +443,79 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>Advertising=</varname></term>
```

Comment:
hmm, i think this should be "Advertise=" rather than "Advertising=". That sounds more along what the english language would use here...